data/wpa_supplicant/wpa_supplicant_test.sh

```diff
@@ -116,9 +116,12 @@ if systemctl is-active wickedd >& /dev/null; then
     echo "Restarting wicked ... "
     systemctl restart wicked
     sleep 20
-    systemctl status wicked
+    systemctl status wicked | cat
```

Comment:
why we are ok if wicked has failed state here ?

Reply:
Good catch, I need to set `pipefail` as well. The reason for this is that otherwise the script blocks when executed outside of openQA

Reply:
I've added `set -o pipefail`, should be ok now. Please mark as resolved if you agree